FHEM/14_SD_Keeloq.pm

```diff
@@ -1117,14 +1115,10 @@ sub SD_Keeloq_binsplit_Roto($) {
 	my $bits = shift;
 	my $binsplit;
 
-	for my $i(0..65){
+	for my $i(0..65) {
```

Comment:
Verstehe ich das richtig, dass Du mit dieser Schleife nur alle 16 Zeichen ein Leerzeichen in den String einbaust und sich der Abschnitt dann ändert?

Reply:
Diese Schleife ist nur zum debugen, weil die Bits sich so wiederfinden in der Hersteller - Schaltkreiserläuterung. Somit sieht man besser durch.